dcs/action.py

```diff
@@ -1787,6 +1787,195 @@ def dict(self):
         return d
 
 
+class PictureAction(Action):
+
+    class HorzAlignment(Enum):
+        Left = "0"
+        Center = "1"
+        Right = "2"
+
+        def __eq__(self, other) -> bool:
+            if isinstance(other, str):
+                return self.value == other
+            return self == other
+
+    class VertAlignment(Enum):
+        Top = "0"
+        Center = "1"
+        Bottom = "2"
+
+        def __eq__(self, other) -> bool:
+            if isinstance(other, str):
+                return self.value == other
+            return self == other
+
+    class SizeUnits(Enum):
+        Original_Size = "0"
```

Comment:
`OriginalSize` or `ORIGINAL_SIZE` (PEP8 prefers the latter, we have a mix, I suppose lean toward PEP8)

Reply:
Noted.